x-pack/plugin/ml/qa/native-multi-node-tests/src/test/java/org/elasticsearch/xpack/ml/integration/MlNativeIntegTestCase.java

```diff
@@ -191,10 +190,6 @@ protected boolean upgradeMode() {
         return client().execute(PutFilterAction.INSTANCE, new PutFilterAction.Request(filter)).actionGet();
     }
 
-    protected GetFiltersAction.Response getMlFilters() {
-        return client().execute(GetFiltersAction.INSTANCE, new GetFiltersAction.Request()).actionGet();
-    }
```

Comment:
Why does this method have to be removed as part of this PR?

Reply:
It doesn't have to be. But it was unused and I was exploring this class for fixing this problem. 

Following the [boy scout rule](https://www.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)